assembl/static2/js/app/utils/translations.js

```diff
@@ -798,7 +799,8 @@ const Translations = {
     },
     date: {
       format: "D MMMM YYYY",
-      format2: "DD-MM-YYYY"
+      format2: "DD-MM-YYYY",
+      format3: "DD/MM/YYYY"
```

Comment:
Really please don't use a new format, the short format should be consistent in all pages. don't follow blindly the maquette.

Reply:
The wireframes are the specifications that we've been communicated so I'll try to follow them as possible, it's our mission. Nevertheless I understand the need to be consistent. For BrightMirror, some layouts are not designed to be consistent to the rest of assembl so those cases might happen again.